tools/PI/DevHome.PI/Strings/en-us/Resources.resw

```diff
@@ -1013,4 +1013,28 @@
     <value>Collapse the large content panel</value>
     <comment>The tooltip for a button which collapses the large content panel.</comment>
   </data>
+  <data name="AppName.ToolTipService.ToolTip" xml:space="preserve">
+    <value>Name of the attached app</value>
+    <comment>Refers to the app's name in the bar</comment>
+  </data>
+  <data name="ClipboardMonitor.ToolTipService.ToolTip" xml:space="preserve">
+    <value>Open the Clipboard Monitor window</value>
+    <comment>Tooltip that tells users what the Clipboard Monitor button will do</comment>
+  </data>
+  <data name="ClipboardMonitorLabel.Text" xml:space="preserve">
+    <value>Clipboard Monitor</value>
+    <comment>A text label for the Clipboard Monitor button</comment>
+  </data>
+  <data name="Insights.ToolTipService.ToolTip" xml:space="preserve">
+    <value>See insights that PI has gained</value>

```

Comment:
I don't think we're supposed to use "PI" in UI.

Reply:
Updated. Thanks!